src/main/java/bio/terra/cli/command/resource/Resolve.java

```diff
@@ -18,7 +18,6 @@
 /** This class corresponds to the third-level "terra resource resolve" command. */
 @Command(name = "resolve", description = "Resolve a resource to its cloud id or path.")
 public class Resolve extends BaseCommand {
-
```

Comment:
Thanks for the help. It seems come from the last time I manually solve the conflict.